test/espnet2/enh/decoder/test_stft_decoder.py

```diff
@@ -58,3 +60,24 @@ def test_STFTDecoder_invalid_type(
         )
         x_lens = torch.tensor([300 * hop_length, 295 * hop_length], dtype=torch.long)
         y, ilens = decoder(real, x_lens)
+
+
+def test_conv_enc_dec_streaming():
```

Comment:
Now I think we need to test the consistency for STFTEncoder and STFTDecoder both separately and jointly.
That is, make sure:
  * the streaming mode of STFTEncoder/STFTDecoder is equivalent to the corresponding normal mode
  * the cascaded STFTEncoder and STFTDecoder can reconstruct the input waveform

Reply:
Done.